source/clear-linux/guides/network/custom-clear-container.rst

```diff
@@ -1,43 +1,42 @@
 .. _custom-clear-container:
 
-Build a custom Clear Linux based Docker container image
-#######################################################
+Build a custom Clear Linux\* based Docker\* container image
+###########################################################
 
-The official base |CLOSIA| container image is published on Docker\* Hub and
-is updated on a regular basis. This section contains the steps to build a
-custom image.
+The official base |CLOSIA| container image is published on Docker\* Hub and is
+updated on a regular basis. This section contains the steps to build a custom
+container image.
 
 Prerequisites
 *************
 
-* These steps must be performed on a |CL| system because the `swupd` command
-  is needed to manage bundles in the container.
-* The `containers-basic` bundle must be installed on the |CL| system for
-  Docker to work.
-* Basic knowledge of Docker is required.
+* You must perform these steps on a |CL| system because the
+  :abbr:`swupd (software updater)` is used to manage bundles in the
+  container.
+* You must install the :file:`containers-basic` bundle on the |CL| system
+  or Docker will not work.
+* You have a basic understanding of Docker.
 
-Build the base Clear Linux container image
-******************************************
+Build the base container image
+******************************
 
 #. Log in and get root privileges.
 
-   .. code-block:: console
+   .. code-block:: bash
 
-      $ sudo -s
+      sudo -s
 
 #. Verify Docker is installed and running.
 
-   .. code-block:: console
+   .. code-block:: bash
 
-      # docker info
+      sudo docker info
```

Comment:
Since the first command you do is to get root privileges using the `sudo -s` command, you don't need to use sudo for the rest of the commands, so you need to remove sudo from the commands listed

Reply:
Thanks for the feedback. I will confirm the changes with @rcaballeromx, perhaps I did too much.

Reply:
QA and I have reached an agreement. 
Please make sure the `sudo -s` prefaces all other commands. 
We will then add the dependency on the test cases so that it gets executed before the other commands, even under parallel execution.
Once the command is in place @MCamp859,  you can remove `sudo` from all other commands.